tests/Pay/Adapter/StripeTest.php

```
@@ -292,6 +292,69 @@ public function testPurchase(array $data): array
+            'number' => '4000000000000341',
+            'exp_month' => 8,
+            'exp_year' => 2030,
+            'cvc' => 123,
+        ]);
+        $this->assertNotEmpty($failingPm['id']);
+        $failingPmId = $failingPm['id'];
+
+        // Create a payment intent with the failing payment method
+        $paymentIntentId = null;
+        try {
+            $purchase = $this->stripe->purchase(5000, $customerId, $failingPmId);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$purchase'. (Unused Code Rules)

(UnusedLocalVariable)